vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -35,6 +35,30 @@ function DownloadFileWithRetry {
     }
 }
 
+function DownloadFileWithAzCopy {
```

Comment:
Use the same func format `Download-FileWithAzCopy`, `Cleanup-TemporaryFiles` and rename `DownloadFileWithRetry` to `Download-File`.
Could you help to try whether we can use `Download-FileWithAzCopy` to instead of `DownloadFileWithRetry` so we can remove `DownloadFileWithRetry` ?

Reply:
I think we can only use `Download-FileWithAzCopy` in `configure-windows-vhd.ps1`.

Reply:
Azcopy could only be used for Azure Blob, Azure Files and Azure Data Lake Storage Gen2 (download files to local). https://learn.microsoft.com/en-us/azure/storage/common/storage-ref-azcopy-copy. So we still need `Download-File`.